app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/vm/operations.rb

```diff
@@ -50,4 +50,32 @@ def raw_remove_all_snapshots
     create_notification(:vm_snapshot_failure, :error => err.to_s, :snapshot_op => "delete")
     raise MiqException::MiqVmSnapshotError, err.to_s
   end
+
+  def remote_console_acquire_ticket_queue(protocol, _userid)
+    task_opts = {
+      :action => "acquiring Instance #{name} #{protocol.to_s.upcase} remote console ticket",
+    }
+
+    queue_opts = {
+      :instance_id => id,
+      :class_name  => self.class.name,
+      :method_name => 'remote_console_acquire_ticket',
+      :priority    => MiqQueue::HIGH_PRIORITY,
+      :role        => 'ems_operations',
+      :args        => [protocol]
+    }
+
+    MiqTask.generic_action_with_callback(task_opts, queue_opts)
+  end
```

Comment:
Follow-up but I think we should move this up to the base class since this method appears to be duplicated in a number of providers.